spec/request/connected_applications_request_spec.rb

```diff
@@ -27,5 +27,25 @@
         end
       end
     end
+
+    it 'returns no app after deleting an app' do
```

Comment:
This is testing that calling index after a delete will not return a cached result.  But after this change https://github.com/department-of-veterans-affairs/vets-api/pull/5420 it's nothing we intend to do.  So I'm going to remove this test but leaving it in this pr history for posterity.